### DISCUSSION TITLE
Possible to add image in table?

### DISCUSSION THREAD

**Discussion by shivam5992** [h2oai/wave#1255] [Q&A]
I want to display an image in the ui.table. 

<img width="701" alt="image" src="https://user-images.githubusercontent.com/4301965/154905127-485de918-3992-45ad-a3a1-5ccd75126b4b.png">

Is it possible in the latest release? Is there any example?

**Comment by mturoci** [comment]
Not possible currently, but could be useful probably. Feel free to open a feature request with all the details you wish to be added.
